src/cpg_seqr_loader/stages.py

```diff
@@ -109,7 +109,7 @@ def queue_jobs(self, multicohort: targets.MultiCohort, inputs: stage.StageInput)
         outputs = self.expected_outputs(multicohort)
 
         job = generate_densify_jobs(
-            input_vds=inputs.as_str(multicohort, CombineGvcfsIntoVds),
+            input_vds=inputs.as_dict(multicohort, CombineGvcfsIntoVds)['vds'],
```

Comment:
```suggestion
            input_vds=inputs.as_str(multicohort, CombineGvcfsIntoVds, 'vds'),
```

quick change to make sure the variable comes back as a String